CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## [1.1.0] - 2022-08-31
+
+- Nothing changed.
```

Comment:
I don't think it's necessary to mention the previous version number, because the changelog always describes changes compared to the previous version. I rephrased it anyway a bit.